packages/toolkit/package.json

```diff
@@ -40,39 +37,25 @@
     "cross-spawn": "^7.0.3",
     "css-loader": "^6.5.1",
     "cssnano": "^5.0.11",
-    "eslint": "^8.0.0",
-    "eslint-config-airbnb": "^19.0.4",
-    "eslint-config-airbnb-base": "^15.0.0",
-    "eslint-config-prettier": "^8.3.0",
-    "eslint-plugin-import": "^2.25.3",
-    "eslint-plugin-jest": "^25.2.4",
-    "eslint-plugin-jsdoc": "^37.0.3",
-    "eslint-plugin-jsx-a11y": "^6.5.1",
-    "eslint-plugin-prettier": "^4.0.0",
-    "eslint-plugin-react": "^7.27.0",
-    "eslint-plugin-react-hooks": "^4.3.0",
-    "eslint-webpack-plugin": "^3.1.1",
     "html-webpack-plugin": "^5.5.0",
     "ignore-emit-webpack-plugin": "2.0.6",
     "image-minimizer-webpack-plugin": "^3.2.3",
     "jest": "^27.3.1",
     "mini-css-extract-plugin": "^2.4.5",
     "minimist": "^1.2.5",
     "postcss": "^8.4.4",
-    "postcss-css-variables": "^0.18.0",
     "postcss-editor-styles": "^0.3.0",
     "postcss-import": "^14.0.2",
     "postcss-loader": "^6.2.0",
     "postcss-mixins": "^8.1.0",
     "postcss-preset-env": "^7.0.0",
-    "prettier": "2.4.1",
```

Comment:
Even with a minor/patch update in `prettier` it can make code that was previously well-formatted and passing eslint fail. Simply because any update in prettier will likely fix a bug in code formatting!.

Therefore the prettier folks recommend using an exact version of prettier.

It could be confusing that simply running `npm install` could update things and thus require code-reformating on a project. For that reason I'm thinking we need to be more explicit with prettier version here.

Additionally, since we're releasing a new major, we can update prettier as well.

Reply:
The goal of 10up-toolkit is to allow users to install only one package and have everything they need, so I'd like to avoid asking users to install packages manually.

I think we can lock in prettier to a specific version, I don't think it is a big deal and we can instruct devs to use `npm` override config if they want to update pretier manually.

Reply:
I agree though that this is not ideal.